util/reggen/gen_dv.py

```diff
@@ -35,9 +35,9 @@ def miname(m):
     return m.name.lower()
 
 
-def sv_base_addr(b):
+def sv_base_addr(b, inst):
     '''Get the base address of a block in SV syntax'''
-    return "{}'h{:x}".format(b.width, b.base_addr)
+    return "{}'h{:x}".format(b.width, b.base_addr[inst])
```

Comment:
Probably throw an exception to cleanly exit if `inst` is not a key of `b.base_addr`.

Reply:
if it's not the right key, it will exit with error to show what key is used, right? Not sure if there is a value to add additional check.

Reply:
It will likely show some ugly Python code trace with an error. It will likely not indicate the key. I guess its fine.

Reply:
I am wrong - it does say `KeyError`, so its not needed.

Reply:
Thanks for confirming that